tools/health/monitorUtil.py

```diff
@@ -46,8 +46,9 @@ def run(cmd, inData="") :
         p = subprocess.Popen(cmd, stdout=subprocess.PIPE, stdin=subprocess.PIPE, stderr=subprocess.STDOUT)
         if (inData != ""):
             p.stdin.write(inData)
-        p.wait()
```

Comment:
not used in open - but used in blue.
we'd have to move either the test from blue here or the scripts there.